src/test/java/org/openmrs/contrib/qaframework/automation/AdvancedInstallSteps.java

```diff
@@ -9,16 +9,16 @@
  */
 package org.openmrs.contrib.qaframework.automation;
 
-import org.openmrs.contrib.qaframework.RunTest;
-import org.openmrs.contrib.qaframework.helper.InitialSetupPage;
-
 import io.cucumber.java.After;
 import io.cucumber.java.Before;
 import io.cucumber.java.en.And;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
 
+import org.openmrs.contrib.qaframework.RunTest;
```

Comment:
is this change neccessary here

Reply:
@sherrif10 I don't remember making changes on that file.